core/storage/blog/gae_models.py

```diff
@@ -334,15 +334,62 @@ def has_reference_to_user_id(cls, user_id):
             cls.editor_ids == user_id).get(keys_only=True) is not None
 
     @classmethod
-    def get_by_user(cls, user_id):
-        """Retrieves the rights object for all blog posts assigned to given user
+    def get_multi_published_models_by_user(cls, user_id, limit=None):
```

Comment:
Don't need "multi" here because "published models" is already plural.

Reply:
Done.